### DISCUSSION TITLE
pnpm install failed when the package has bundledDependencies

### DISCUSSION THREAD

**Discussion by balanceXLR** [pnpm/pnpm#5605] [Q&A]
1. i published a npm package by `npm publish`
2. call `pnpm install` 
3. throw error  ERR_PNPM_FETCH_404  GET https://registry.npmjs.org/@xxx Not Found - 404

**Comment by lvqq** [comment]
Any concrete case to reproduce it? If your npm package is installed via pnpm, you should use `node-linker=hoisted` before you publish it: https://pnpm.io/npmrc#node-linker
